packages/toolkit/package.json

```diff
@@ -704,7 +704,7 @@
                 {
                     "id": "aws.amazonq.codewhisperer",
                     "name": "%AWS.amazonq.codewhisperer.title%",
-                    "when": "!isCloud9 && !aws.isSageMaker && !aws.toolkit.amazonq.dismissed && !aws.explorer.showAuthView"
```

Comment:
Why was this changed? I don't see the `setContext` value being changed. This is not a function call.
See https://github.com/aws/aws-toolkit-vscode/blob/5e8ad316718661bed35230e3596c13e5e2b53f30/packages/core/src/extensionNode.ts#L121-L126

Reply:
Good catch! That was a mistake. I believe I removed the whole clause and then attempted to undo it. Fixed it.